Fix systemd renewer script

elyscape opened the pull request:
This corrects the systemctl command. Additionally, it simplifies the shell script used and switches to sh, which unlike bash is guaranteed to be available.

See also smallstep/certificates#748.

maraino reviewed (approved):
Looks good to me. Just letting you know that `.OragnizationalData` is more secure because it's a value that is controlled in the ca.json

While .Subject.CommonName is coming from the token or CSR.

elyscape committed:
Fix systemd renewer script

This corrects the systemctl command. Additionally, it simplifies the
shell script used and switches to sh, which unlike bash is guaranteed to
be available.

tashian commented:
Fixed in fd31fcb
